### PR TITLE
Tweak: Expanding existing area for editing columns, sections and containers [HTS-55]

### PR DESCRIPTION
**Pull request by mark-elementor**
## PR Checklist

- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by nivnoimanelementor — DISMISSED** [review]
Looks good.

File: assets/dev/scss/frontend/_global.scss [hunk 1]
```diff
@@ -122,6 +122,14 @@
 			background-size: cover;
 		}
 	}
+
+	.elementor-click-area {
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
```
Comment (StyleShit) [inline assets/dev/scss/frontend/_global.scss:130]:
```suggestion
		inset: 0;
```

File: includes/elements/column.php [hunk 2]
```diff
@@ -919,6 +919,7 @@ protected function content_template() {
 		?>
 		<div class="elementor-<?php echo esc_attr( $wrapper_element ); ?>-wrap">
 			<div class="elementor-background-overlay"></div>
+			<div class="elementor-edit-click-area"></div>
```
Comment (StyleShit) [inline includes/elements/column.php:922]:
```suggestion
			<div class="e-edit-click-area"></div>
```
Maybe? (also others)

File: includes/elements/container.php [hunk 3]
```diff
@@ -114,6 +114,7 @@ protected function get_initial_config() {
 	 */
 	protected function content_template() {
 		?>
+		<div class="elementor-edit-click-area"></div>
```
Comment (StyleShit) [inline includes/elements/container.php:117]: Does the click area works also on first Editor load?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/_global.scss [hunk 4]
```diff
@@ -122,6 +122,14 @@
...
+		position: absolute;
```
Comment (ManorHazaz) [inline assets/dev/scss/frontend/_global.scss:131]: inset: 0; ?

**Review by ManorHazaz — CHANGES_REQUESTED** [review]

File: assets/dev/scss/frontend/_global.scss (hunk 1 above)
Comment (mark-elementor) [reply assets/dev/scss/frontend/_global.scss]: Done

File: includes/elements/column.php (hunk 2 above)
Comment (mark-elementor) [reply includes/elements/column.php]: Good idea, renamed them all

File: includes/elements/container.php (hunk 3 above)
Comment (mark-elementor) [reply includes/elements/container.php]: It does

**Review by StyleShit — DISMISSED** [review]

**Review by nivnoimanelementor — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**